rest/pagination/api.graphql

```diff
@@ -0,0 +1,118 @@
+# This example shows how @rest is configured for pagination.
+#
+# @rest supports three styles of pagination for REST responses:
+#
+# PAGE_NUMBER - pages by requesting a specific page number
+# OFFSET - pages by reqesting a starting record offset
+# NEXT_CURSOR - pages by opaque cursors returned by the REST API
+#
+# In each case @rest allows the backend REST API to be exposed
+# in the GraphQL schema as a field using
+# GraphQL Cursor Connections Specification.
+# https://relay.dev/graphql/connections.htm
+
+"""
+Sample customer type.
+"""
+type Customer {
+  id: ID!
+  name: String
+  email: String
+}
+
+"""
+`CustomerConnection` is the connection type for `Customer` pagination.
+A field returning a connection type must have arguments
+`first` (number of nodes to fetch) and `after` (opaque cursor indicating
+the starting point).
+"""
+type CustomerConnection {
+  edges: [CustomerEdge]
+  pageInfo: PageInfo!
+}
+
+"""
+`CustomerEdge` provides access to the node and its cursor.
```

Comment:
Stopped review here, but wanted to get what I have so far to you

Reply:
Thanks!